CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing
+
+The documentation for each repository is available in the "**docs/**" directory of that repository.
+It is written in [Markdown format] and built using [MkDocs].
+
+## Testing Your Changes Locally
+
+We **strongly** encourage you to test your changes locally before contributing them.
+To do so, you first need to do three things; these are:
+
+1. Have MkDocs installed.
+2. Have the Laminas documentation theme available to the laminas-form directory.
+3. Update `mkdocs.yml` to build the docs using the documentation theme.
+
+To complete these steps, please follow the instructions below.
+
+1. [Install MkDocs].
+2. Install [the PyMdown Extensions].
+3. Clone the [documentation-theme repository] locally.
+    ```
+    git clone git@github.com:laminas/documentation-theme.git documentation-theme
+    ```
+4. Copy the theme directory from the cloned documentation-theme repository to the root directory of this project.
+5. Add the following Yaml snippet to `mkdocs.yml`, changing it to suit the repository that you are working with.
+    ```yaml
+    # The following example assumes that you are working with laminas-form
+    repo_url: 'https://github.com/<your-project>'
+    extra:
+        repo_name: <your-project>
+        project: Components
```

Comment:
A notice should be added that "Mezzio" and "MVC" is also an option, related to component itself. Maybe after the code example.

Reply:
Thanks for the feedback.

Reply:
What about Windows, which doesn't support symlinks?

Reply:
I stand corrected. I just found out that versions Windows 7, 8, & 10 (I love counting in Microsoft) do support symlinks.

Reply:
I added in a Yaml comment above the project line, because that way the comment would be much closer to the relevant line, rather than below the code block.